data/json/vehicle_parts.json

```diff
@@ -5334,5 +5334,27 @@
         "breaks_into" : [
             {"item": "plastic_chunk", "min": 1, "max": 2}
         ]
+    },
+    {
+        "type" : "vehicle_part",
+        "id" : "vehicle_scoop",
+        "name" : "scoop",
+        "symbol" : "R",
+        "color" : "light_gray",
+        "broken_symbol" : ";",
+        "broken_color" : "dark_gray",
+        "durability" : 100,
+        "epower" : -50,
+        "size":400,
```

Comment:
I've tried to compile your branch and got this error:
![untitled](https://cloud.githubusercontent.com/assets/5421954/9152421/524b7a1a-3e30-11e5-94fd-5fd6fd578ee5.png)
The reason of such behavior is you put both ```size``` & ```bonus``` together. I didn't check if you're actually use these values - please remove unused one / both of them if they're not necessary.